File: LEGDS/fun2-1.4/java/Main.java
import java.util.Scanner;
import src.ArraySum;

public class Main{

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.printf("Please enter a double value%n");
		double x = scan.nextDouble();
		System.out.printf("Please enter number of sum%n");
		int n = scan.nextInt();

		System.out.printf("Result of sum is %.2f %n",ArraySum.arraySum(x, n));
		System.out.printf("Recursion is %.2f %n",ArraySum.arraySum0(x,n));

		scan.close();
	}
}

File: LEGDS/fun2-1.4/c/main.c
#include "main.h"

int main() {
	int n;
	double x;

	printf("Please enter a double value\n");
	scanf("%lf",&x);
	printf("Please enter number of sum\n");
	scanf("%d",&n);

	printf("Result of sum is %.2lf\n",arraySum(x, n));
	printf("Recursion is %.2lf \n",arraySum0(x,n));
	return 0;
}
